src/services/api/repositories/petition.js

```diff
@@ -23,6 +23,11 @@ export default {
   update: (petition) => {
     const requestPath = path.join('/petitions', petition.id.toString());
     delete petition.id;
+
+    if (petition.city && !petition.city.data) {
+      delete petition.city.data;
+    }
```

Comment:
And this one? Maybe this was an API change… If you pass a null object as the data property to the city on a form you‘re updating, the request fails as you need to pass an empty object/nothing but not null.